fdbserver/EncryptKeyProxy.actor.cpp

```diff
@@ -805,7 +898,12 @@ ACTOR Future<Void> getLatestBlobMetadata(Reference<EncryptKeyProxyData> ekpProxy
 					dbgTrace.get().detail("BMI" + std::to_string(item.domainId), "");
 				}
 			}
+			ekpProxyData->kmsConnectorHealthStatus = KMSHealthStatus{ true, true, now() };
```

Comment:
we still want to check if response `vector.size > 1` before updating the KmsHealthStatus?